### PR TITLE
use `floating-point` in docs

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *use `floating-point` in docs*

### Tickets:
 - *59113*

### CODE REVIEW

**Review by rkazants — APPROVED** [review]

**Review by mitruska — APPROVED** [review]

**Comment by pelszkow** [comment]
PR related with https://github.com/openvinotoolkit/openvino/pull/6608#discussion_r669400836

**Comment by pelszkow** [comment]
@ilyachur, RP looks ready to be merge.

File: docs/ops/activation/Sigmoid_1.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ sigmoid( x ) = \frac{1}{1+e^{-x}}
 
 **Inputs**:
 
-*   **1**: Input tensor *x* of any floating point type. **Required.**
+*   **1**: Input tensor *x* of any floating-point type. **Required.**
 
 **Outputs**:
 
-*   **1**: Result of Sigmoid function applied to the input tensor *x*. Floating point tensor with shape and type matching the input tensor.
+*   **1**: Result of Sigmoid function applied to the input tensor *x*. floating-point tensor with shape and type matching the input tensor.
```
Comment (jdanieck) [inline docs/ops/activation/Sigmoid_1.md:27]: Need to capitalize here.

Comment (pelszkow) [reply docs/ops/activation/Sigmoid_1.md:27]: Good catch. I will recheck it.

Comment (pelszkow) [reply docs/ops/activation/Sigmoid_1.md:27]:
```suggestion
*   **1**: Result of Sigmoid function applied to the input tensor *x*. Floating-point tensor with shape and type matching the input tensor.
```

Comment (pelszkow) [reply docs/ops/activation/Sigmoid_1.md]: Fortunately it looks there is only one case with capital letter here.

**Review by ilyachur — APPROVED** [review]

**Pull request merged**